src/build/build-workflow.ts

```diff
@@ -95,6 +96,12 @@ export interface BuildWorkflowOptions {
    * @default "{ pullRequest: {}, workflowDispatch: {} }"
    */
   readonly workflowTriggers?: Triggers;
+
+  /**
+   * Permission that are required for the job
+   * @default `{contents: 'write'}`
+   */
+  readonly workflowPermissions?: JobPermissions;
```

Comment:
This one should still be called `permissions` only the one in release needs to workflow prefix.

```suggestion
  readonly permissions?: JobPermissions;
```